src/plugins/index_pattern_management/public/service/creation/index.ts

```diff
@@ -8,3 +8,5 @@
 
 export { IndexPatternCreationConfig, IndexPatternCreationOption } from './config';
 export { IndexPatternCreationManager } from './manager';
+// @ts-ignore
```

Comment:
Rollups had two `js` files that I'd prefer not to convert to typescript at this point. There are plans for rollup improvements that would make these files unnecessary.